### PR TITLE
Update Communications SDK doc for GA.

### PR DESCRIPTION
**Pull request by zihzhan**

### CODE REVIEW

File: Samples/V1.0Samples/LocalMediaSamples/PolicyRecordingBot/README.md [hunk 1]
```diff
@@ -60,35 +60,41 @@ To verify your policy was assigned correctly:
...
-1. Publish the bot from VS:
+4. Publish the bot from VS:
     1. Right click PolicyRecordingBot, then click `Publish...`. Publish it to the cloud service you created earlier.
 
 ### Test
 
 1. Set up the test meeting and test clients:
    1. Sign in to Teams client with a non-recorded test tenant user.
-   1. Use another Teams client to sign in with the recorded user. (You could use an private browser window at https://teams.microsoft.com)
+   2. Use another Teams client to sign in with the recorded user. (You could use an private browser window at https://teams.microsoft.com). Use Teams client if you couldn't see the call notification by using Teams browser for some reason.
```
Comment (ananmishr) [inline Samples/V1.0Samples/LocalMediaSamples/PolicyRecordingBot/README.md:85]: Maybe rephrase number 2 as
Use another Teams client to sign in with the recorded user. You could use an private browser window and open up https://teams.microsoft.com. If the call notification doesn't appear on web, use the Teams desktop client

File: Samples/V1.0Samples/LocalMediaSamples/PolicyRecordingBot/README.md [hunk 2]
```diff
@@ -60,35 +60,41 @@ To verify your policy was assigned correctly:
...
 
-1. Place a call from the Teams client with the non-recorded user to the recorded user.
+2. Place a call from the Teams client with the non-recorded user to the recorded user.
 
-1. Your bot should now receive an incoming call, and join the call (See next step for retrieving logs). Use the recorded user's Teams client to accept the call.
+3. Your bot should now receive an incoming call, and join the call (See next step for retrieving logs). Use the recorded user's Teams client to accept the call. A banner should show up once the call get established.
```
Comment (ananmishr) [inline Samples/V1.0Samples/LocalMediaSamples/PolicyRecordingBot/README.md:89]: Rephrase this as

Your bot should receive the incoming call. Use the recorded users' Teams client to accept the call. Once the call is established, you'll see a banner indicating that the recording has started. See the next step to learn how you can retrieve the call log.

File: Samples/V1.0Samples/LocalMediaSamples/PolicyRecordingBot/README.md (hunk 1 above)
Comment (zihzhan) [reply Samples/V1.0Samples/LocalMediaSamples/PolicyRecordingBot/README.md:85]: Good Phrase, I'll take it.

File: Samples/V1.0Samples/LocalMediaSamples/PolicyRecordingBot/README.md (hunk 2 above)
Comment (zihzhan) [reply Samples/V1.0Samples/LocalMediaSamples/PolicyRecordingBot/README.md:89]: Change it to 
```
Your recording bot should receive the incoming call and join the call immediately. Use the recorded users' Teams client to accept the call. Once the P2P call is established, you'll see a banner indicating that the recording has started. See the next step to learn how you can retrieve the call log.
```
